### PR TITLE
Update WS+Emit.swift

### PR DESCRIPTION
**Pull request by tierracero**
This update will handle events with no payload with key or event as event trigger on text mode

### CODE REVIEW

File: CodyFire/Classes/WS/WS+Emit.swift [hunk 1]
```diff
@@ -51,6 +51,10 @@ extension WS {
         let key: String
     }
```
Comment (MihaelIsaev) [inline CodyFire/Classes/WS/WS+Emit.swift:52]: Please also delete `EM` models since it is not needed after your fix

**Review by tierracero — COMMENTED** [review]
Remove  EM structs

File: CodyFire/Classes/WS/WS+Emit.swift [hunk 2]
```diff
@@ -47,50 +47,20 @@ extension WS {
...
     
-    private struct EM: Encodable {
-        let key: String
-    }
-    
     private struct GM: Encodable {
         let event: String
     }
     
-    public func emit<Model: WSEventModel>(data event: WSEventIdentifier<Model>, completion: (() -> Void)? = nil) {
```
Comment (MihaelIsaev) [inline CodyFire/Classes/WS/WS+Emit.swift:54]: Please leave it `(data event` to be consistent with another method which is declared on line 32.

**Review by MihaelIsaev — APPROVED** [review]

**Review by tierracero — COMMENTED** [review]
rename GM -> SimpleEvent
and added emit  data simple event

**Comment by MihaelIsaev** [comment]
@tierracero thank you for the contribution! 🚀 Your changes are available since `1.15.3`

**Pull request merged**